.github/workflows/dcr-lighthouse.yml

```diff
@@ -41,6 +41,9 @@ jobs:
           npm install -g puppeteer-core@2.1.0 @lhci/cli@0.10.0
           lhci autorun
 
+      # We're running the deployment workflow on "push" events so that it can be used in both PRs and on the main branch.
+      # "push" event doesn't give us access to the usual PR info so we need to use an action in order to obtain that information.
+      # This action works by using the Github search API to find a associated PR given a commit ID.
```

Comment:
Should we not run on two types of events? [`pull_request`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) and [`push` for main](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-to-specific-branches-occurs)?

Reply:
I recall having a good reason for not using pull_request but I can't remember what it was for the life of me!